bundler.d/assets.rb

```diff
@@ -1,7 +1,7 @@
 group :assets do
   gem 'sass-rails', '~> 3.2'
   gem 'uglifier', '>= 1.0.3'
-  gem 'execjs', '>= 1.4.0'
+  gem 'execjs', '>= 1.4.0', '<2.5.0'
```

Comment:
Shouldn't this be `< 2.5.1` then? I don't understand execjs concept of semantic versioning though if they require a different major Ruby version for a patch version of their project :confused: